airflow/executors/celery_executor.py

```diff
@@ -41,6 +41,8 @@
 
 CELERY_SEND_ERR_MSG_HEADER = 'Error sending Celery task'
 
+OPERATION_TIMEOUT = conf.getint('celery', 'operation_timeout')
```

Comment:
Can we add a test for this? Let's say a user change this setting to `conf.get('celery', 'operation_timeout')` that test should fail.

Also let's add a fallback value.

```
OPERATION_TIMEOUT = conf.getint('celery', 'operation_timeout', fallback=2)
```

Reply:
Thanks, @kaxil. Done. I added a very boring test like this. It'll detect mistakes such as the example you showed. Pls let me know if you are looking for something better.

```python
def test_operation_timeout_config():
    assert celery_executor.OPERATION_TIMEOUT == 2
```